src/style/Header.module.css

```diff
@@ -1,19 +1,58 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500;600;700;800&display=swap');
 
+.list {
+  height: 500px;
+  overflow-y: scroll;
+}
+
+/* Books.js styling */
+.list::-webkit-scrollbar {
+  display: none;
+}
+
+.loading {
+  height: 500px;
```

Comment:
- [ ] Please make sure the add book section starts on the same line vertically as the book container so please add left margins to it. Please also fix the categories and add book buttons.

### Your design

![image](https://github.com/Dorcas126/Book-store-project/assets/83803414/09e164c0-32a0-406e-a6aa-3f9a7307adee)

### Zeplin design

![image](https://github.com/Dorcas126/Book-store-project/assets/83803414/768b275c-1cb6-4e20-88f5-f2bb6edc1be7)